CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+## 0.11.0 (2022/09/19)
```

Comment:
just noticed this date will need to change, I believe we are hard fixed on the 0.11.0 release date so it should be safe to add here, but maybe confirm with @sswati0  or @prsekar